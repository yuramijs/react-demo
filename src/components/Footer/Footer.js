import React from 'react';
import withStyles from 'isomorphic-style-loader/lib/withStyles';
import s from './Footer.scss';

import FooterLogoColor from './footer-logo-color.png';

const Footer = () =>
  <div className={s.footer}>
    <img className={s.footer__logo} src={FooterLogoColor} alt="Adnami footer logo"/>
  </div>;

export default withStyles(s)(Footer);
