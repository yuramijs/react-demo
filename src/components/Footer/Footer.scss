.footer {
  &__logo {
    width: 100%;
    max-width: 95px;
    margin: 10px;
  }
}