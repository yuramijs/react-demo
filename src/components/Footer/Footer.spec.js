import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Footer from './Footer.js';

Enzyme.configure({adapter: new Adapter()});

describe('Footer', () => {
  it('should be defined', () => expect(Footer).toBeDefined());
  it('should render correctly', () => {
    const tree = shallow(
      <div>
        <img class="Footer-footer__logo-3BzF-" src="/assets/src/components/Footer/footer-logo-color.png?220e400b"
             alt="Adnami footer logo"/>
      </div>
    );
    expect(tree).toMatchSnapshot();
  });
});