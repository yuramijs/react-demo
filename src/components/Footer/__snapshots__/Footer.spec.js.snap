// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <div>
    <img
      alt="Adnami footer logo"
      class="Footer-footer__logo-3BzF-"
      src="/assets/src/components/Footer/footer-logo-color.png?220e400b"
    />
  </div>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        alt="Adnami footer logo"
        class="Footer-footer__logo-3BzF-"
        src="/assets/src/components/Footer/footer-logo-color.png?220e400b"
      />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "Adnami footer logo",
        "class": "Footer-footer__logo-3BzF-",
        "src": "/assets/src/components/Footer/footer-logo-color.png?220e400b",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="Adnami footer logo"
          class="Footer-footer__logo-3BzF-"
          src="/assets/src/components/Footer/footer-logo-color.png?220e400b"
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Adnami footer logo",
          "class": "Footer-footer__logo-3BzF-",
          "src": "/assets/src/components/Footer/footer-logo-color.png?220e400b",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
